app/lib/credentialDisplay/openBadgeCredential.tsx

```diff
@@ -35,24 +36,38 @@ export const openBadgeCredentialDisplayConfig: CredentialDisplayConfig = {
 function OpenBadgeCredentialCard({ rawCredentialRecord, onPressIssuer }: CredentialCardProps): React.ReactElement {
   const { styles, theme } = useDynamicStyles(dynamicStyleSheet);
   const { credential } = rawCredentialRecord;
-  const { credentialSubject, issuer, name } = credential;
+  const { credentialSubject, issuer } = credential;
 
   const issuanceDate = getIssuanceDate(credential);
   const expirationDate = getExpirationDate(credential);
   const formattedIssuanceDate = issuanceDate ? moment(issuanceDate).format(DATE_FORMAT) : 'N/A';
   const formattedExpirationDate = expirationDate ? moment(expirationDate).format(DATE_FORMAT) : 'N/A';
 
   const {
+    issuedTo,
     description,
     criteria,
-    subjectName,
     numberOfCredits,
     startDateFmt,
     endDateFmt,
     achievementImage,
     achievementType
   } = credentialSubjectRenderInfoFrom(credentialSubject);
 
+  const isOpenBadgeCredential = credential.type.includes('OpenBadgeCredential');
+
+  const issuedto = issuedTo?.trim() 
```

Comment:
In this code, we're using two different variables `issuedto` and `issuedTo` that differ only in lowercase/uppercase. This is generally bad practice. 
Also, this logic belongs not here, in the view component, but in the `utils/credentialSubject.ts` file below.